ledger/src/builtins.rs

```diff
@@ -11,37 +11,33 @@ macro_rules! to_builtin {
 }
 
 /// Builtin programs that are always available
-fn genesis_builtins(cluster_type: ClusterType, bpf_jit: bool) -> Vec<Builtin> {
-    let mut builtins = vec![
+fn genesis_builtins(bpf_jit: bool) -> Vec<Builtin> {
+    vec![
         to_builtin!(solana_bpf_loader_deprecated_program!()),
         if bpf_jit {
             to_builtin!(solana_bpf_loader_program_with_jit!())
         } else {
             to_builtin!(solana_bpf_loader_program!())
         },
-    ];
-    if cluster_type != ClusterType::MainnetBeta {
```

Comment:
`cluster_type != ClusterType::MainnetBeta` was what was breaking testnet specifically.   But more generally it's not necessary to introduce cluster-specific programs here at all.  The programs in `feature_builtins()` get enabled automatically when `ClusterType::Development` and for all other cluster types the feature must be first activated